app/models/vm_synchronize.rb

```diff
@@ -18,7 +18,7 @@ def call_synchronize
 
     options[:categories] ||= Vm.default_scan_categories
     begin
-      host = Host.find(agent_id)
+      host = MiqServer.find(agent_id)
```

Comment:
@roliveri The reference in the automate spec is part of the `excluded_model_names` variable.  The test is checking that all sub-classed models have automate service_model counterparts.  However, there are a few models we want to exclude.  Since `VmSynchronize` is a sub-class of the `Job` model it shows up in the check.

If the `VmSynchronize` model is removed the test reference can be removed as well.